src/Emulator.cpp

```diff
@@ -60,18 +60,9 @@ void Emulator::Start()
 
 void Emulator::Stop()
 {
-    // Free SDL resources
-    if (m_renderer != nullptr)
-    {
-        SDL_DestroyRenderer(m_renderer);
-        m_renderer = nullptr;
-    }
-
-    if (m_window != nullptr)
-    {
-        SDL_DestroyWindow(m_window);
-        m_window = nullptr;
-    }
+    m_cpu.reset();
```

Comment:
We could do m_cpu = nullptr, but generally .reset() is the accepted way to release a unique_ptr.